module.js

```diff
@@ -13,14 +16,11 @@ const cachedTextDecoder = new TextDecoder("utf-8", {
 
 cachedTextDecoder.decode();
 
-// let cachedUint8Memory0;
-// function getUint8Memory0() {
-//   if (cachedUint8Memory0.byteLength === 0) {
-//     cachedUint8Memory0 = new Uint8Array(wasm.memory.buffer);
-//   }
-//   return cachedUint8Memory0;
-// }
-
+/**
```

Comment:
These docs don't have descriptions because these are all internal APIs, the docs are just here for the types.